doc/installation.rst

```diff
@@ -54,15 +62,33 @@ We recommend using this installation method on those platforms, but we offer sev
 .. _Windows Installer: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder_64bit_full.exe
 .. _macOS Installer: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder.dmg
 
+.. note::
+
+   "Lite" versions of both installers are also available from the `releases page`_, which are somewhat smaller than the full installers.
+   These lack a number of optional but recommended dependencies, such as NumPy, SciPy and Pandas, meaning that full :doc:`/panes/variableexplorer` support for NumPy arrays and Pandas DataFrames will not be available, along with advanced data import and a few other features (you'll still be able to use them in your own Python environments, so long as you've installed them).
+   Given this only saves a modest amount of space while missing out on significant features, we recommend using the full installers unless minimizing download/install size and memory usage is your highest priority.
```

Comment:
Not sure we need to mention this. Seems like we are saying they are not worth it and if they are not then why do we have them. What do you think?

Reply:
On the issue/elsewhere, Carlos/Daniel mentioned explicitly steering users toward full installers unless they have a reason to use the lite version, which makes sense. However, they are "worth it" for specific cases, as the above explains,

> [where] minimizing download/install size and memory usage is your highest priority.

If I was running Spyder on a particularly old or lower-performance device, as a number of our users do (for instance, I myself and other users have run it on Raspberry Pis) and don't need the Variable Explorer features, that could potentially save up to several hundred mb of disk space and memory, as well as decrease its surface area for potential bugs, which may make the difference in being able to use it or not.

Reply:
I think I made the suggestion wrong. I meant the part starting from "Given this..." doesn't seem necessary

Reply:
Hmm, I didn't see any suggestion to begin with, which would have been very helpful in highlighting specifically what you were requesting. Do you mean the entire sentence starting with "Given this", or a specific part of it? The former is what I initially guessed you were referring to, and what I responded to above.

Reply:
For the sake of clarity in this discussion:
* the full version is 725MB and the lite version is 405MB (as of 5.1.5)
* AFAIK, the lite version has limiting features only related to Variable Explorer
* Given our intent to isolate Spyder code from user's environments, I think that in the future the lite version may become the official version if/when:
  * the functionality of the Variable Explorer is not hindered
  * Spyder has its own package/environment manager

Of course the last point is far from certain, as it is a long-term development which may change or even be rejected.

I do think, at least for now, we should be steering users to the full version. It may be worth discussing with @spyder-ide/core-developers whether we should be creating the lite version right now. What do most users need? Is having two versions too confusing for most users? Is there a sufficient number of users of the lite version to warrant it?

When I first made the standalone application, I created both version primarily because I did not require the full version. However, I may not represent most users 😄 . I have also been considering that we should reserve creating the lite version only for releases, not for every pull-request. I.e. pull requests only create the full version (useful for full feature testing) and releases create both full and lite versions.

Reply:
Thanks for the details and confirmation, @mrclary . I've revised this section somewhat to simplify and clarify the description as a response to your and @juanis2112 's comments.

Its certainly worth discussing whether the lite version is needed, though I think it might be best to do that in another venue (like a spyder-ide/sppyder issue) rather than a review comment thread on this near-final PR on Spyder-Docs.

Reply:
> Its certainly worth discussing whether the lite version is needed, though I think it might be best to do that in another venue (like a spyder-ide/sppyder issue) rather than a review comment thread on this near-final PR on Spyder-Docs.

Absolutely, I only mentioned it here for completeness and in case it was relevant to your efforts.

@CAM-Gerlach , thank you very much for your hard work revising and updating the documentation. I know how much work goes into this (I do quite a bit of documentation for my job at times), only to have it outdated so quickly 😄 .

Reply:
Thanks for your kind words, @mrclary . The lion's share of the credit for that goes to @juanis2112 , she's the one who does the really substantial updates, especially of the screenshots and GIFs that take a lot of time to re-do and need to be updated frequently given the constant UI changes in Spyder. I mostly just do reviews (which even that is now partially taken away from me, albeit forcibly imposed), minor updates/fixes and behind the scenes stuff (backend/infra/CI maintenance, technical changes, etc), and even that I was AWOL from for most of the past three months, and then caused no end of problems when I returned.

Incidentally, we were just speaking about possibly implementing a monthly "release" system for those types of changes to reduce the heavy load on her for that.

Reply:
In any case, @mrclary , do you think what we have here is good enough? If so, feel free to mark this as resolved, thanks.